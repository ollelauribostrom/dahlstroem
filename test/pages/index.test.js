import React from 'react';
import { shallow } from 'enzyme';

import IndexPage from '../../src/pages/index';

const mockData = {
  datoCmsFrontpage: {
    backgroundImage: 'img',
    color: { hex: '#fff' },
    menuBackground: { hex: '#fff' },
    menuColor: { hex: '#000' },
  },
};

describe('Tests for Index page', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<IndexPage data={mockData} />);
    expect(wrapper).toMatchSnapshot();
  });
});
