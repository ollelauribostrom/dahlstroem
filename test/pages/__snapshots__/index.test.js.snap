// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for Index page renders correctly 1`] = `
<Layout
  color={
    Object {
      "hex": "#fff",
    }
  }
  menuBackground={
    Object {
      "hex": "#fff",
    }
  }
  menuColor={
    Object {
      "hex": "#000",
    }
  }
  path="/test"
  width={810}
>
  <SEO
    keywords={
      Array [
        "Felix DahlstrÃ¶m Persson",
        "Portfolio",
      ]
    }
    lang="en"
    meta={Array []}
    title="Home"
  />
  <ImageBackground
    src="img"
  />
</Layout>
`;
